### ISSUE TITLE
paired end reads overlap

### ISSUE THREAD

**Issue by hmyh1202** [zwdzwd/biscuit#48]
Hello:

  Dose the biscuit  only consider one copy base when paired end reads overlap ?

thank you

**Comment by zwdzwd** [comment]
Yes, it should. Please use https://github.com/huishenlab/biscuit as the official place for discussion btw.
